docs/tutorials/validate_data.rst

```diff
@@ -152,20 +316,43 @@ to stop or issue a warning in the code:
   else:
     logger.warning("This file is not a valid batch of {0:s}".format(str(data_asset_name)))
 
+The ``validation_result`` object has detailed information about every expectation in the suite that was used to validate the batch: whether the batch met the expectation and even more details if it did not. You can read more about the result object's structure here: :ref:`validation_result`.
+
+You can print this object out:
+
+.. code-block:: python
+
+    print(json.dumps(validation_result, indent=4))
+
+
+Here is what a part of this object looks like:
+
+.. image:: ../images/validation_playground_result_json.png
+    :width: 500px
+
+Don't worry - this blob of JSON is meant for machines. :ref:`data_docs` provide an HTML view of both expectation suites and validation results that is more suitable for humans. You will see how to get to them in the next sections.
+
 
-Validation Operators
---------------------
+7. Validation Operators
+-----------------------
 
-Validation Operators and Actions make it possible to define collections of tasks together that should be done after a
-validation. For example, we might store results (either on a local filesystem or to S3), send a slack notification,
-and update data documentation. The default configuration performs each of those actions. See the
+The `validate` method evaluates one batch of data against one expectation suite and returns a dictionary of validation results. This is sufficient when you explore your data and get to know Great Expectations.
+When deploying Great Expectations in a real data pipeline, you will typically discover additional needs:
+
+* validating a group of batches that are logically related
+* validating a batch against several expectation suites
+* doing something with the validation results (e.g., saving them for a later review, sending notifications in case of failures, etc.).
```

Comment:
```suggestion
* Validating a group of batches that are logically related (eg Did all my salesforce integrations work last night?).
* Validating a batch against several expectation suites (eg Does my nightly clickstream event job have any **critical** failures I need to deal with asap or **warnings** I should investigate later?).
* Doing something with the validation results (e.g., saving them for a later review, sending notifications in case of failures, etc.).
```

Reply:
+1